test/powershell/Host/TabCompletion/BugFix.Tests.ps1

```diff
@@ -54,4 +54,15 @@ Describe "Tab completion bug fix" -Tags "CI" {
            $result.CompletionMatches[1].CompletionText | Should Be $DatetimeProperties[1].Name # DateTime
        }
     }
+    It "Issue#3373 - '`$error[0].<tab>' should work" {
+        try {
+            5/0
+        }
+        catch {
+            $e = $error[0]
+        }
+        $cmd = "5/0;`$error[0]."
+        $result = TabExpansion2 -inputScript $cmd -cursorColumn $cmd.Length
+        $result.CompletionMatches.Count | Should Be ($e | Get-Member).Count
+    }
```

Comment:
All tests above checks `CompletionText`. Maybe add that for `$cmd = "5/0;`$error[0].FullyQ` ?